select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2Choice.java

```diff
@@ -77,7 +77,13 @@ public Select2Choice(String id, ChoiceProvider<T> provider)
 		super(id, provider);
 	}
 
-	@Override
+    // needed to be able override the class on Scala
```

Comment:
I don't remember having such problem in my Scala apps.
This would be a major issue in Scala if it is true.
I will work on a Scala app in the coming days and I'll verify this.
Having this method is not a big deal but I'd prefer to remove it. Or at least document why it has to be there.